index.html

```diff
@@ -1770,6 +1848,28 @@
             default presentation.
           </div>
         </section>
+        <section>
+          <h4>
+            Receiving user agent
+          </h4>
+          <p>
+            The following partial interface MUST be implemented by a
+            <a>receiving user agent</a>:
+          </p>
+          <pre>
+          partial interface Presentation {
+            [SameObject]
+            readonly attribute PresentationReceiver? receiver;
+          };
+          
+</pre>
+          <p>
+            The <dfn for="Presentation"><code>receiver</code></dfn> MUST return
+            <code>null</code> if the current <a>browsing context</a> is not a
+            <a>receiving browsing context</a> and return a
+            <a><code>PresentationReceiver</code></a> instance otherwise.
+          </p>
```

Comment:
> Can the current browsing context not be a receiving browsing context for a receiving user agent?

I believe the implication is that all browsing contexts on the receiving UA would have `navigator.presentation.receiver` defined but set to null if the context is not a presentation.  @mounirlamouri I believe you added this definition?

> defaultRequest will always return null in a receiving user agent that is not a controlling user agent as well.

What if script attempts to set this property - will it still return null?  That seems a little odd.
Perhaps we should hide the property instead?

> receiver will always return null in a controlling user agent that is not a receiving user agent as well.

Done.

Your proposal in #3 and #4 seems to prevent a single user agent from implementing both the controlling and  receiving browsing contexts!  This puts Chrome or any other 1-UA mode browser in a tough situation :-)